app/controllers/provider_interface/application_choices_controller.rb

```diff
@@ -1,5 +1,7 @@
 module ProviderInterface
   class ApplicationChoicesController < ProviderInterfaceController
+    before_action :set_application_choice
```

Comment:
Should not apply to `index` (or revert this controller change).

Reply:
fixed